rocAL/source/readers/file_source_reader.cpp

```diff
@@ -58,16 +58,9 @@ Reader::Status FileSourceReader::initialize(ReaderConfig desc) {
     _shuffle = desc.shuffle();
     _loop = desc.loop();
     _meta_data_reader = desc.meta_data_reader();
+    _last_batch_info = desc.get_last_batch_policy();
+    _pad_last_batch = _last_batch_info.second; // Currently last_batch_padded and pad_last_batch are same
```

Comment:
Lets name this last_batch_padded and not confused with pad_last_batch comment added here

Reply:
This variable seems unused. Please remove it

Reply:
Remove from the .h file too